userspace/libpman/src/capture.c

```diff
@@ -0,0 +1,126 @@
+#include "state.h"
+#include <scap.h>
+
+void libpman__enable_capture()
+{
+	g_state.skel->bss->g_settings.capture_enabled = true;
+}
+
+void libpman__disable_capture()
+{
+	g_state.skel->bss->g_settings.capture_enabled = false;
+}
+
+#ifdef TEST_HELPERS
+
+/* Not used right now */
+int libpman__print_stats()
+{
+	char error_message[MAX_ERROR_MESSAGE_LEN];
+	long overall_n_drops_buffer = 0;
+	long overall_n_drops_max_event_size = 0;
+	long overall_n_evts = 0;
+	struct counter_map cnt_map;
+
+	int counter_maps_fd = bpf_map__fd(g_state.skel->maps.counter_maps);
+	if(counter_maps_fd <= 0)
+	{
+		libpman__print_error("unable to get counter maps");
+		return errno;
+	}
+
+	for(int index = 0; index < g_state.n_cpus; index++)
+	{
+		if(bpf_map_lookup_elem(counter_maps_fd, &index, &cnt_map) < 0)
+		{
+			sprintf(error_message, "unbale to get the counter map for CPU %d", index);
+			libpman__print_error((const char *)error_message);
+			goto clean_print_stats;
+		}
+		overall_n_evts += cnt_map.n_evts;
+		overall_n_drops_buffer += cnt_map.n_drops_buffer;
+		overall_n_drops_max_event_size += cnt_map.n_drops_max_event_size;
+	}
+	printf("\noverall_n_evts: %ld\n", overall_n_evts);
```

Comment:
Being this a library, just wondering if we should do an `sprintf` on an output string instead?

Reply:
This method is just for local testing purposes, it's not used in userspace code or in tests, it could be used just to print some statistics, probably it's easier to use with direct prints inside it, WDYT about it?